pkg/controlplane/instance.go

```diff
@@ -263,11 +263,12 @@ func (c *Config) createLeaseReconciler() reconcilers.EndpointReconciler {
 	endpointsAdapter := reconcilers.NewEndpointsAdapter(endpointClient, endpointSliceClient)
 
 	ttl := c.ExtraConfig.MasterEndpointReconcileTTL
-	config, err := c.ExtraConfig.StorageFactory.NewConfig(api.Resource("apiServerIPInfo"))
```

Comment:
I'm surprised to see this change... this isn't a REST API, so I wouldn't expect it to be encryptable

Reply:
reverted